java/ql/lib/ext/kotlin.model.yml

```diff
@@ -3,5 +3,7 @@ extensions:
       pack: codeql/java-all
       extensible: summaryModel
     data:
+      - ["kotlin", "StandardKt", False, "apply", "", "", "Argument[0]", "Argument[1].Parameter[0]", "value", "manual"]
+      - ["kotlin", "StandardKt", False, "apply", "", "", "Argument[0]", "ReturnValue", "value", "manual"]
```

Comment:
This looks wrong. Shouldn't this be `Argument[1].ReturnValue` instead of `Argument[0]`?

Reply:
Huh!? Maybe not, I guess. A quick googling of "kotlin apply" does seem to indicate that its semantics are somewhat conflicting with what I expect from a function called "apply".

Reply:
This seems to confirm it as well: https://kotlinlang.org/docs/scope-functions.html#apply